webdriver/tests/support/helpers.py

```diff
@@ -151,10 +151,7 @@ def center_point(element):
 
 
 def document_hidden(session):
-    """Polls for the document to become hidden."""
-    def hidden(session):
-        return session.execute_script("return document.hidden")
-    return Poll(session, timeout=3, raises=None).until(hidden)
```

Comment:
Seems like we had this poll because of rare intermittent failures mentioned at https://bugzilla.mozilla.org/show_bug.cgi?id=1492499#c3

I see you've pushed to try, but we should retrigger the test jobs to be confident this is no longer a problem

Reply:
I did a lot of retriggers for https://treeherder.mozilla.org/jobs?repo=try&revision=de2740755f3343d222caf5b849610378b507e317 and this is no longer an issue.